workbench/.htaccess

```diff
@@ -1,6 +1,14 @@
 # The following settings are optional, additional configurations for an Apache Web Server.
 # This file may be deleted if Workbench is hosted in an environment that does not allow .htaccess overrides.
 
+# add custom header to all requests
```

Comment:
Do we need the following two lines? They just seem like comments.

Reply:
oops, I left those in by mistake, thanks for pointing it out. I removed them in my latest commit.